modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/pagemem/BPlusTreeReuseListPageMemoryImplTest.java

```diff
@@ -119,7 +119,8 @@
                 }
             },
             () -> true,
-            new DataRegionMetricsImpl(new DataRegionConfiguration(), cctx.metric(), NO_OP_METRICS),
+            new DataRegionMetricsImpl(
+                new DataRegionConfiguration(), cctx.metric(), cctx.performanceStatistics(), NO_OP_METRICS),
```

Comment:
Should we add `PerformanceStatistics` instance to ctx on line 68?

Reply:
done